### PR TITLE
XMAS-4195 - Fix future warning and bump version

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Description:
- Fixed a future warning around a None check
- Bumped version to 0.1.7

#### Testing:
- Create a viewer and verify that the future warning is fixed

**review**:
@Workiva/xmas

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='ixbrl_viewer',
-    version='0.1.6',
+    version='0.1.7',
```
Comment (sagesmith-wf) [inline setup.py]: FYI: `os.getenv("GIT_TAG")` might be a good option in the future

Comment (brettkail-wk) [reply setup.py]: Opened [XMAS-4198](https://jira.atl.workiva.net/browse/XMAS-4198).

**Comment by brianmyers-wf** [comment]
QA +1
verified warning is gone from local testing
@Workiva/release-management-p

**Pull request merged**